src/main/java/uk/gov/digital/ho/proving/income/hmrc/domain/IncomeRecord.java

```diff
@@ -8,28 +8,32 @@
 
 import java.time.LocalDate;
 import java.util.List;
-import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toList;
 
 @AllArgsConstructor
 @Getter
 @Accessors(fluent = true)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class IncomeRecord {
 
-    @JsonProperty
+    @JsonProperty(value = "paye", required = true)
```

Comment:
What's the motivation for adding the value="paye" etc. when that's the name of the variable also? I generally prefer explicit over implicit but I'm not convinced this adds any value, especially as it's not done consistently throughout the code.